### PR TITLE
77264 capitalisation

### PR DESCRIPTION
**Pull request by stevelee-civica**
change content capitalisation

### CODE REVIEW

File: src/EA.Weee.Domain/Lookup/WeeeCategory.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public enum WeeeCategory
         [Display(Name = "Lighting equipment")]
         LightingEquipment = 5,
 
-        [Display(Name = "Electrical and electronic tools")]
+        [Display(Name = "electrical and electronic tools")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Domain/Lookup/WeeeCategory.cs:22]: I don't think this one should be changed as it should be capitalised at the start of the sentence?

File: src/EA.Weee.Domain/Organisation/Organisation.cs [hunk 2]
```diff
@@ -53,7 +53,7 @@ private set
             {
                 if (value != null && (value.Length < 7 || value.Length > 15))
                 {
-                    throw new InvalidOperationException("Company registration number must be 7 to 15 characters long");
+                    throw new InvalidOperationException("company registration number must be 7 to 15 characters long");
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Domain/Organisation/Organisation.cs:56]: Think this should be capitalised, as its start of sentence.

File: src/EA.Weee.Web/Areas/Admin/ViewModels/AddOrganisation/Details/RegisteredCompanyDetailsViewModel.cs [hunk 3]
```diff
@@ -16,8 +16,8 @@ public class RegisteredCompanyDetailsViewModel : OrganisationViewModel
         public override string BusinessTradingName { get; set; }
 
         [Required(ErrorMessage = "Enter company registration number (CRN)")]
-        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The Company registration number should be 7 to 15 characters long")]
-        [Display(Name = "Company registration number (CRN)")]
+        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The company registration number should be 7 to 15 characters long")]
+        [Display(Name = "company registration number (CRN)")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/ViewModels/AddOrganisation/Details/RegisteredCompanyDetailsViewModel.cs:20]: Don't think this one should be changed, the one in the sentence looks ok.

File: src/EA.Weee.Web/Areas/Admin/ViewModels/Organisation/EditRegisteredCompanyOrganisationDetailsViewModel.cs [hunk 4]
```diff
@@ -12,8 +12,8 @@ public class EditRegisteredCompanyOrganisationDetailsViewModel : EditOrganisatio
         public string CompanyName { get; set; }
 
         [Required(ErrorMessage = "Enter company registration number (CRN)")]
-        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The Company registration number should be 7 to 15 characters long")]
-        [Display(Name = "Company registration number (CRN)")]
+        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The company registration number should be 7 to 15 characters long")]
+        [Display(Name = "company registration number (CRN)")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/ViewModels/Organisation/EditRegisteredCompanyOrganisationDetailsViewModel.cs:16]: Don't think this one should be changed, the one in the sentence looks ok.

File: src/EA.Weee.Web/Areas/Admin/Views/Aatf/Details.cshtml [hunk 5]
```diff
@@ -274,7 +274,7 @@
                 @if (Model.Organisation.OrganisationType == EA.Weee.Core.Organisations.OrganisationType.RegisteredCompany)
                 {
                     <tr class="govuk-table__row">
-                        <th class="govuk-table__header" scope="row">Company registration number (CRN)</th>
+                        <th class="govuk-table__header" scope="row">company registration number (CRN)</th>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/Views/Aatf/Details.cshtml:277]: Dont think this should change as its heading.

File: src/EA.Weee.Web/Areas/Admin/Views/Producers/_detailsResults.cshtml [hunk 6]
```diff
@@ -78,7 +78,7 @@
                     @if (@scheme.CompanyNumber != null)
                     {
                         <tr class="govuk-table__row">
-                            <th class="govuk-table__header" scope="row">Company registration number</th>
+                            <th class="govuk-table__header" scope="row">company registration number</th>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/Views/Producers/_detailsResults.cshtml:81]: Dont think this should change as its heading.

File: src/EA.Weee.Web/Areas/Admin/Views/Scheme/Overview/RegisteredCompanyDetailsOverview.cshtml [hunk 7]
```diff
@@ -25,7 +25,7 @@
                         <td class="govuk-table__cell">@Model.CompanyName</td>
                     </tr>
                     <tr class="govuk-table__row">
-                        <th class="govuk-table__header" scope="row">Company registration number (CRN)</th>
+                        <th class="govuk-table__header" scope="row">company registration number (CRN)</th>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Admin/Views/Scheme/Overview/RegisteredCompanyDetailsOverview.cshtml:28]: Dont think this should change as its heading.

File: src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditOrganisationDetails.cshtml [hunk 8]
```diff
@@ -50,7 +50,7 @@
         </div>
         <div class="govuk-grid-row govuk-!-padding-bottom-2">
             <div class="flex-container">
-                <span class="govuk-body govuk-label--s label-width" for="Organisation_CompaniesRegistrationNumber" id="Organisation_CompaniesRegistrationNumber-label">Company registration number</span>
+                <span class="govuk-body govuk-label--s label-width" for="Organisation_CompaniesRegistrationNumber" id="Organisation_CompaniesRegistrationNumber-label">company registration number</span>
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditOrganisationDetails.cshtml:53]: Dont think this should change as its heading.

**Review by JohnPadgettCivica — COMMENTED** [review]
@stevelee-civica  added some comments. Where the items are at the start of the sentence they should be capital.

File: src/EA.Weee.Domain/Lookup/WeeeCategory.cs (hunk 1 above)
Comment (stevelee-civica) [reply src/EA.Weee.Domain/Lookup/WeeeCategory.cs]: restored

File: src/EA.Weee.Domain/Organisation/Organisation.cs (hunk 2 above)
Comment (stevelee-civica) [reply src/EA.Weee.Domain/Organisation/Organisation.cs:56]: restored

File: src/EA.Weee.Web/Areas/Admin/ViewModels/Organisation/EditRegisteredCompanyOrganisationDetailsViewModel.cs (hunk 4 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Admin/ViewModels/Organisation/EditRegisteredCompanyOrganisationDetailsViewModel.cs:16]: restored

File: src/EA.Weee.Web/Areas/Admin/Views/Aatf/Details.cshtml (hunk 5 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Admin/Views/Aatf/Details.cshtml:277]: restored

File: src/EA.Weee.Web/Areas/Admin/Views/Producers/_detailsResults.cshtml (hunk 6 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Admin/Views/Producers/_detailsResults.cshtml:81]: restored

File: src/EA.Weee.Web/Areas/Admin/Views/Scheme/Overview/RegisteredCompanyDetailsOverview.cshtml (hunk 7 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Admin/Views/Scheme/Overview/RegisteredCompanyDetailsOverview.cshtml:28]: restored

File: src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditOrganisationDetails.cshtml (hunk 8 above)
Comment (stevelee-civica) [reply src/EA.Weee.Web/Areas/Producer/Views/ProducerSubmission/EditOrganisationDetails.cshtml:53]: restored

File: src/EA.Weee.Core/Organisations/Base/OrganisationViewModel.cs [hunk 9]
```diff
@@ -25,8 +25,8 @@ public class OrganisationViewModel : IValidatableObject
 
         public ExternalAddressData Address { get; set; } = new ExternalAddressData() { CountryId = UkCountry.Ids.England };
 
-        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The Company registration number should be 7 to 15 characters long")]
-        [Display(Name = "Company registration number (CRN)")]
+        [StringLength(maximumLength: EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber, MinimumLength = 7, ErrorMessage = "The company registration number should be 7 to 15 characters long")]
+        [Display(Name = "company registration number (CRN)")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Core/Organisations/Base/OrganisationViewModel.cs:29]: @stevelee-civica  this one need reverting?

File: src/EA.Weee.Web.Tests.Unit/Areas/Admin/ViewModels/EditRegisteredCompanyOrganisationDetailsViewModelTests.cs [hunk 10]
```diff
@@ -51,13 +51,13 @@ public void GivenModel_CompaniesRegistrationNumberShouldHaveStringLengthAttribut
                 .BeDecoratedWith<StringLengthAttribute>(a => a.MaximumLength.Equals(EnvironmentAgencyMaxFieldLengths.CompanyRegistrationNumber)
                                                              && a.MinimumLength.Equals(7)
                                                              && a.ErrorMessage.Equals(
-                                                                 "The company registration number should be 7 to 15 characters long"));
+                                                                 "The Company registration number should be 7 to 15 characters long"));
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web.Tests.Unit/Areas/Admin/ViewModels/EditRegisteredCompanyOrganisationDetailsViewModelTests.cs:54]: @stevelee-civica  this one is ok as it is in a sentence not at the start of a sentence.

**Pull request merged**